DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -72,6 +74,7 @@ public class CnCNetGameLobby : MultiplayerGameLobby
                 new NoParamCommandHandler(MAP_SHARING_DISABLED_MESSAGE, HandleMapSharingBlockedMessage),
                 new NoParamCommandHandler("RETURN", ReturnNotification),
                 new IntCommandHandler("TNLPNG", TunnelPingNotification),
+                new IntCommandHandler("PNGCHK", PingCheckNotification),
```

Comment:
On a second thought - is it really needed to implement another command? Perhaps it would be better (as in less code and no additional messages sent, so less chance of being kicked for flood) if each client would just broadcast their pings?